src/next-app/cypress/e2e/shopRegistration.cy.ts

```diff
@@ -0,0 +1,15 @@
+describe('create shop owner account and shop', () => {
+  it('should create shop owner account', () => {
+    cy.visit('/create-account');
+    
+    cy.get('input[id="First Name"]').type('John');
+    cy.get('input[id="Last Name"]').type('Smith');
+    cy.get('input[id="Phone Number"]').type('1123456789');
+    cy.get('input[id="Email"]').type('test@email.com');
+    cy.get('input[id="Username"]').type(`SHOP-${Math.round(Math.random()*10000)}`);
+    cy.get('input[id="Password"]').type('ShopPass123');
+    cy.get('input[id="Type"]').type('Shop').get('button[id="Shop Owner"]').click();

```

Comment:
This `click()` method for some reason is not triggering the "click" event in JavaScript. So, we can manually override this by doing this:

```javascript
    cy.get('input[id="Type"]').click(); // open the dropdown menu
    cy.get('button[id="Shop Owner"]').then(($button) => {
      const button = $button[0]; // get the DOM node
      button.dispatchEvent(new Event('click', { bubbles: true, cancelable: true }));
    });
```